metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -342,6 +369,13 @@ public void rename(
             .includeDefinitionMetadata(true)
             .includeDataMetadata(true)
             .build()).orElseThrow(() -> new TableNotFoundException(oldName));
+
+        // Fail if the table is tagged not to be renamed.
+        if (hasTags(oldTable, config.getNoTableRenameOnTags())) {
+            throw new IllegalArgumentException(
+                String.format("Table %s cannot be renamed because it is tagged.", oldName));
+        }
```

Comment:
adding the tag that fails this in the message?

Reply:
Will add.